### ISSUE TITLE
Add diagnostic for overriden method name case mismatch

### ISSUE THREAD

**Issue by jakubmisek** [peachpiecompiler/peachpie#657]
In php overriden method can have a different letter casing.

Compiler handles this case by generating a valid CLR override stub and introducing a new method. This causes unnecessary overhead, also it is a sign of bad coding..
```php
class A { function foo() {} }
class B extends A {
  function Foo() {} // <-- here
}
```
An info diagnostic should warn user about this and suggest a quick fix.

- [x] INF_OverrideNameCaseMismatch
- [x] docs
